tests/helper_test.py

```diff
@@ -143,13 +143,13 @@ def _test(self, base, expected):
         self.assertListEqual(actual, expected)
 
     def test_bytes(self):
-        base = b'echo "foo bar"'
-        expected = [b'echo', b'foo bar']
+        base = 'echo "foo bar"'
+        expected = ['echo', 'foo bar']
         self._test(base, expected)
 
     def test_unicode(self):
-        base = u'echo "foo €"'
-        expected = [b'echo', u'foo €'.encode('utf-8')]
+        base = 'echo "foo €"'
+        expected = ['echo', 'foo €']
```

Comment:
Why is there no unicode string (`u'...'`) in `test_unicode`.  I just checked and it seems to me that `u'foo'` is just a plain `str`. Is that always true in py3?  In some cases the `u` prefix has been removed from literals but in some cases it has not. Is there any meaning behind this?

Reply:
In python 3 a raw string `'string'` is always unicode (or str), and byte strings must be decorated, `b'bytes'`, python 2.x is the opposite. The reason the u prefix is removed in some cases but not others is that 2to3 removes it, but on the files that 2to3 didn't handle well I didn't always remove them.